packages/core-ethereum/src/account.ts

```diff
@@ -58,9 +58,9 @@ class Account {
 
     this._nonceTracker = new NonceTracker({
       getLatestBlockNumber: async () => {
-        // tests fail if we don't query the block number
-        // when running on ganache
-        return !coreConnector.network || coreConnector.network === 'localhost'
+        // when running our unit/intergration tests using ganache,
+        // the indexer doesn't have enough time to pick up the events and reduce the data
+        return isGanache(coreConnector.network)
```

Comment:
Not sure whether we should use this for our integration tests but it is fine for now

Reply:
we have no choice - this is needed for our "pseudo" unit tests, am simply turning this into a utility so it's not copy pasted